### DISCUSSION TITLE
Idea: Show Relationships in the Dependency view

### DISCUSSION THREAD

**Discussion by edhans** [TabularEditor/TabularEditor3#1111] [Ideas]
It would be nice when I select a field in a table and look at the dependencies, if it would also tell me it is participating in a relationship in addition to other objects like measures or calculated columns/tables.

**Comment by mlonsk** [comment]
If I understand this correctly this feature has now been in TE3 for a while ont he dependency view "Relationships starting from this column"
<img width="923" height="861" alt="image" src="https://github.com/user-attachments/assets/2573936f-8459-4da4-9055-fd0cf29b69d0" />
Appologies for never updating this idea when the improvement was created.
